pkg/storage/bolt.go

```diff
@@ -105,9 +105,8 @@ func (b *BoltDB) ReadPrefix(namespace, prefix string) (map[string][]byte, error)
 	err := b.db.View(func(tx *bolt.Tx) error {
 		bucket := tx.Bucket([]byte(namespace))
 		if bucket == nil {
-			errMsg := fmt.Sprintf("namespace<%s> does not exist", namespace)
-			logrus.Error(errMsg)
-			return errors.New(errMsg)
+			logrus.Infof("namespace<%s> does not exist", namespace)
```

Comment:
I don't think we should log on this since it's expected behavior.

Instead, a test can detect whether the implementation is correct. Is there a unit test covering this behavior?

Reply:
Yup agree, 

I have created unit tests and removed this log